### ISSUE TITLE
[TBC The Eye] - Curse of Tongues or Mind-numbing Poison Not Slowing High Astromancer Solarian's Arcane Missiles

### ISSUE THREAD

**Issue by Cooties-cc** [chromiecraft/chromiecraft#6838]
### What client do you play on?

enGB

### Faction

Both

### Content Phase:

70

### Current Behaviour

When Curse of Tongues is cast on High Astromancer Solarian, the cast time still remains at 3 seconds long.

[CoT.webm](https://github.com/chromiecraft/chromiecraft/assets/62584589/661b1503-69c2-456d-85e9-8571e75c1ed3)

### Expected Blizzlike Behaviour

The curse mods _casting speed_, not spell haste, thus should increase the casting time from 3 seconds to ~3.9 seconds. This would spread the ticks of the cast to 1.3 seconds, up from 1 every second. 

### Source

https://www.wowhead.com/wotlk/spell=11719/curse-of-tongues
![image](https://github.com/chromiecraft/chromiecraft/assets/62584589/c10bb076-3eaa-4222-85fc-9dc73bc14e6b)

### Steps to reproduce the problem

1. Enter TK and fight Solarian
2. Test cast speed with and w/o debuff
3. They are the same cast timers

### Extra Notes

https://github.com/chromiecraft/chromiecraft/issues/4413#issuecomment-1722705169
Talked about how bosses are not "immune" to be affected by such spells, and that there's alternative proof that -cast speed does work. There's also to say that in PvP, a mage's arcane missiles are affected by CoT, going from every 1 second, to 1.3 seconds per tick. Overall cast time up to 6.5 from 5. 

To put into perspective the -dps on the boss this does, if every tick from boss does 4200 damage (unmitigated) which is every second, that's 4.2k dps per cast. Increasing that to just 1.3 seconds per tick per cast, reduces it to 3230 dps per cast. It's a near -1k dps per cast on boss, which is somewhat significant. 

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/b9c1dc95dd8f150a77673d9f23235b235dd700e4

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by amed80** [comment]
I wonder if this is intended (as i didn't find anything that deny or confirm slowness on Solarain's Arcane Missiles) because Solarian is immune to [Mind-numbing Poison](https://wowgaming.altervista.org/aowow/?item=5237) (slow casting by 30%) when she is not immune to other poisons like [Instant Poison](https://wowgaming.altervista.org/aowow/?item=21927) or [Deadly Poison](https://wowgaming.altervista.org/aowow/?item=22054)

**Comment by Cooties-cc** [comment]
>because Solarian is immune to [Mind-numbing Poison](https://wowgaming.altervista.org/aowow/?item=5237)

![image](https://github.com/chromiecraft/chromiecraft/assets/62584589/9812c575-1d50-4afe-bbfd-1c309ad61ce8)
She doesn't appear to be immune to MNP on PTR, so perhaps something was fixed to correct that issue? Still stands that -30% isn't working, by either spell.

**Comment by amed80** [comment]
Tried looking through multiple Solarian runs pre and post nerf , couldn't get any video that shows Arcane Missiles was ever slowed down also i checked both wowhead and wowgaming for Solarian's [Arcane Missiles](https://wowgaming.altervista.org/aowow/?spell=39414) Cast time :
it says instant cast which in comparison with the [Arcane Missiles](https://wowgaming.altervista.org/aowow/?spell=42846) that Mages use it says Channeled instead.
![image](https://github.com/chromiecraft/chromiecraft/assets/8395873/b3e53335-6682-4d3c-80ed-6ebbdb31289f) 
![image](https://github.com/chromiecraft/chromiecraft/assets/8395873/33de652b-8af2-4b71-9f79-fd98a6012a83)
the Curse will slow the [Void Bolt](https://wowgaming.altervista.org/aowow/?spell=39329) that Solarian uses when she switches form.
Tried to use Curse of Tongues on Shade of Aran in Kara , it have the same effect , will slow both Fire and Frostbolt but it will not slow his Arcane Missiles 5 sec channeling.

**Side Note** : i noticed that Shade of Aran does 1 sec casting his Arcane Missiles then he does the channeling for 5 sec which i think that's not suppose to happen as it's also marked in both sites as Instant Cast and never seen in any Karazhan TBC runs video that i have checked.

**Comment by Cooties-cc** [comment]
Looking at another DB, wowheads wotlk one, it gets a little odd I guess depending on what they are giving. The boss spell [can be seen here](https://www.wowhead.com/wotlk/spell=39414/arcane-missiles#used-by-npc). If you see, it looks similar to the altervista DB posted in data, in that it says "instant". However if you read the other parts, it says stuff like "requires mage", which is odd, and if you check [what triggers it ](https://www.wowhead.com/wotlk/spell=33031/arcane-missiles), it now looks exactly the same as a mage's arcane missiles with a _3 second channel cast time_. For reference, here was the [mage's lvl 70 AM spell](https://www.wowhead.com/wotlk/spell=38704/arcane-missiles). It too looks the same as the other DB, however [checking on the spell effect #2 ](https://www.wowhead.com/wotlk/spell=38703/arcane-missiles), it now looks like Solarians being "instant". 

My guess is that both these DBs might have problems, but it also doesn't explain that no guides/posts ever mention CoT or the poison on the boss. I could factor it to being perhaps scripted different due to it being TBC and not WOTLK? Yet still feels off. As mentioned in the ref to other ticket, all those boss spells had cast timers, and none were labeled as instant. This leads me to think that somehow, Solarians _channeled spell_ is being treated as if it were an _instant cast_, thus any -cast speed debuffs wouldn't affect that spell. Seems dumb to me overall, perhaps a lapse in the script logic, or intentional as a cheap work around to be immune to -cast speed debuffs. 

There's not really any Wotlk era vids doing this boss, at least none that'd really show the debuff. I'm not sure if this would go towards staff choice to make a change, or just leave it as is for a more TBC centric experience. Since there's no hard proof it should work this way, I can't ask for any change tbh.

**Comment by amed80** [comment]
Yeah the only mentions about using Mind Numbing or CoT in that fight was to be used on the Priests that spawn with Solarian.
[Rogue comment](https://www.mmo-champion.com/threads/611981-Rogues-more-useless-than-ever-in-PVE-Really-disappointed?p=5669368&viewfull=1#post5669368)
Also found out that Shade of Aran should be immune to CoT and Mind Numbling which i think is not the case in CC (at least in PTR)

**Comment by balleny** [comment]
keep in mind many abilities could be applied to mobs and bosses without actually taking effect or only partially taking effect.
The cast time or duration of spellcasts can be very rarely influenced afaik.

**Comment by FrancescoBorzi** [comment]
what is the status of this?

**Comment by TheSCREWEDSoftware** [comment]
I can't find a video where I see either the tounges or numb being used for this boss, I only see it being used while clearing the room, but i would assume even if these can be applied as many buffs, they wouldn't take affect
